src/main/webapp/app/shared/competency-selection/competency-selection.component.ts

```diff
@@ -66,6 +69,16 @@ export class CompetencySelectionComponent implements OnInit, ControlValueAccesso
     // halfway between medium and high
 
     ngOnInit(): void {
+        // it's an explicit design decision to don't clutter every component that uses this component with the need to check if the atlas profile is enabled
+        this.profileService.getProfileInfo().subscribe((profileInfo) => {
+            const atlasEnabled = profileInfo.activeProfiles.includes(PROFILE_ATLAS);
+            if (atlasEnabled === true) {
```

Comment:
```suggestion
            if (atlasEnabled) {
```
If I'm not wrong `atlasEnabled` is never undefined. So I aggree with the usually stupid rabbit, that this is a little nicer.